jvm/workbookapp/src/test/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/utils/Audio.kt

```diff
@@ -0,0 +1,18 @@
+package org.wycliffeassociates.otter.jvm.workbookapp.utils
+
+import io.reactivex.Observable
+import org.wycliffeassociates.otter.common.audio.AudioFile
+import org.wycliffeassociates.otter.common.recorder.WavFileWriter
+import java.io.File
+
+object Audio {
+    fun writeWavFile(target: File) {
```

Comment:
utility functions don't need to be inside an object, companion object, or class. They can just exist as functions.